task6_1.php

```diff
@@ -0,0 +1,192 @@
+foreach ($arr as $elem){};
+echo '<a href="/"> главная страница </a></br>';
+if($reqestPatch[1] === 'task6_1.php'){  // первоночальная страница не равна REQUEST_URI
+    echo '<table border="1">';
+    // for ($i =0 ; $i<= 4; $i++){          // c  автоматическим созданием таблицы и ее автозаполнением не срослось
+    //     echo '<tr>' . $arr['Name'][1];
+    //     for ($j =0 ; $j<= 4; $j++){
+    //             echo  '<td>' . $arr['ID'][$j] . '</td>';
+    //             echo  '<td>' . <a href="task6_1.php/ivan">' . $arr['Login'][$j] . '</td>'; // как здесь  подставлять динамичные ссылки?
+    //             echo  '<td>' . $arr['Name'][$j] . '</td>';
+    //               echo  '<td>' . $arr['Surname'][$j] . '</td>';
+    //     }
+    // }
+    echo '<tr>' . '<td>' . $arr['ID'][0] . '</td>' . 
+                  '<td>' . '<a href="task6_1.php/ivan">' . $arr['Login'][0] . '</a>' . '</td>' . // переход по ссылкам не перекрестный а последовательный, идет наслоение, нужен аналог "event.stopPropagation"
+                  '<td>' . $arr['Name'][0] . '</td>' .
+                  '<td>' . $arr['Surname'][0] . '</td>' . 
+                    
+        '</tr>';
+    echo '<tr>' . '<td>' . $arr['ID'][1] . '</td>' . 
+                  '<td>' . '<a href="task6_1.php/tatyana">' . $arr['Login'][1] . '</a>' . '</td>' .   
+                  '<td>' . $arr['Name'][1] . '</td>' .
+                  '<td>' . $arr['Surname'][1] . '</td>' .          
+        '</tr>';
+    echo '<tr>' . '<td>' . $arr['ID'][2] . '</td>' . 
+                  '<td>' . '<a href="task6_1.php/roman">' . $arr['Login'][2] . '</a>' . '</td>' .    
+                  '<td>' . $arr['Name'][2] . '</td>' .
+                  '<td>' . $arr['Surname'][2] . '</td>' .          
+        '</tr>';
+    echo '<tr>' . '<td>' . $arr['ID'][3] . '</td>' . 
+                  '<td>' . '<a href="task6_1.php/evgeniy">' . $arr['Login'][3] . '</a>' . '</td>' .   
+                  '<td>' . $arr['Name'][3] . '</td>' .
+                  '<td>' . $arr['Surname'][3] . '</td>' .          
+        '</tr>';            
+    echo '</table>';
+    exit;
+}
+
+if($reqestPatch[1] === 'task6_1.php/ivan'){ // воспринимает как и просто task6_1.php, не отображает изменения
+    echo '<table border="1">';
+    echo '<tr>' . 
+        '<td>' . $arr['ID'][0] . '</td>' . 
+        '<td>' . '<a href="task6_1.php/ivan">' . $arr['Login'][0] . '</a>' . '</td>' .  
+        '<td>' . $arr['Name'][0] . '</td>' .
+        '<td>' . $arr['Surname'][0] . '</td>' . 
+        '<td>' . $arr['Patronymic'][0] . '</td>' . 
+        '<td>' . $arr['Date of birth'][0] . '</td>' . 
+        '<td>' . $arr['date of registration'][0] . '</td>' . 
+        '<td>' . $arr['date of last account login'][0] . '</td>' . 
+        // '<td>' . '<img src="./public/first.jpg"' . '</td>' .   // как вставить ссылку из массива в текст?   
+        '<td>' . $arr['file name for avatar'][0] . '</td>' . // или так оно подтянет файл?
```

Comment:
```suggestion
        '<td>' . "<img src='" . $arr['file name for avatar'][0] . "'>" . '</td>' . // или так оно подтянет файл?
```

Сработает после того, когда в `$arr['file name for avatar'][0]` будет действительно лежать имя файла.
Либо, если массив будет переформирован согласно прошлому комментарию, то `$users[(int) $reqestPatch[3]]['fileNameOfAvatar']`